_posts/2023-02-01-solidity-0.8.18-release-announcement.md

```diff
@@ -0,0 +1,46 @@
+
+### Compiler Features
+
+* A new command-line flag (`--no-cbor-metadata`) and Standard JSON option (`settings.metadata.appendCBOR: false`) to prevent compiler from appending the CBOR metadata section at the end of the bytecode. Until now it was only possible to exclude the hash of the JSON metadata file (`solc --metadata-hash none`/`settings.metadata.bytecodehash: "none"`) from that section, but the bytecode would still include a small bit of CBOR data containing the version of the compiler.
+* The support to the Paris hard fork includes: 
+    * Deprecation of global `block.difficulty` and removal of `difficulty()` in inline assembly for EVM versions >= Paris. The change is due to the renaming and semantic changes of the `difficulty` instruction introduced by [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399).
+    * Introduction of global `block.prevrandao` in Solidity and `prevrandao()` in inline assembly for EVM versions >= Paris.
+
+*Note*: `block.difficulty` is planned to be completely removed in Solidity version `0.9.0`.
+
+## Full Changelog
+
+#### Important fixes
+*
+*
+
+### Bugfixes
+* Docs build crash with sphinx-a4doc >= 1.4.0 #13820
+* Required boost version in the docs does not match the code after recent changes #13870
+* AST source locations for named mapping keys. #13864
+* 
+
+
+We especially thank all the contributors that made this release possible:
+LIST
+
+To upgrade to the latest version of the Solidity Compiler, simply follow the [installation instructions](https://docs.soliditylang.org/en/v0.8.18/installing-solidity.html) available in our documentation. 
+Our team has made sure to provide detailed and straightforward steps to make the upgrade process as seamless as possible. If you have any questions or run into any issues during the upgrade process, don't hesitate to reach out to our [community Matrix channel](https://matrix.to/#/#ethereum_solidity:gitter.im). 
+You can download the new version of Solidity here: [v0.8.18](https://github.com/ethereum/solidity/releases/tag/v0.8.18).
+If you want to perform a source build, please only use [`solidity_0.8.18.tar.gz`](https://github.com/ethereum/solidity/releases/download/v0.8.18/solidity_0.8.18.tar.gz) and not the zip provided by GitHub directly.
+Please note that for those using Solidity versions below 0.8.0, [there are breaking changes](https://docs.soliditylang.org/en/v0.8.17/080-breaking-changes.html?highlight=update). We highly recommend reviewing the detailed list of breaking changes in our documentation to ensure a smooth upgrade process. Additionally, we encourage all Solidity users to regularly check for updates to stay up-to-date with the latest improvements and optimizations.
+We encourage all Solidity developers to upgrade to version 0.8.18 to take advantage of these improvements and optimizations. Our team remains dedicated to continuously enhancing Solidity to benefit the entire Ethereum ecosystem.
```

Comment:
```suggestion
We recommend all Solidity developers to upgrade to version 0.8.18 to take advantage of these improvements and optimizations. Our team remains dedicated to continuously enhancing Solidity to benefit the entire Ethereum ecosystem.
```
Because we encourage all solidity users to...` was just used in the last sentence.